examples/graphql-context/graphql_context.server.out

```diff
@@ -1 +1 @@
-bal run graphql_context.bal
+$ bal run graphql_context.bal
```

Comment:
Do we need this `$` sign at the beginning? IIRC, we removed this some time back since it makes it harder to copy the command and paste it.

Reply:
That copying issue is fixed now and we need it to differentiate the command from the output in the `.out` file.